app/src/NativeBase/Containers/ProfileContainer.tsx

```diff
@@ -1,73 +1,116 @@
 /**
- * @file App configuration settings.
+ * @file Profile screen showing app configuration settings (and later probably the user's profile).
+ *
+ * The user's dark mode preference (which NativeBase calls colour mode) is handled in two ways:
```

Comment:
After a lot of investigation (and not finding much beyond the basics in NativeBase's docs!) this seems like the best approach to me (but open to alternatives if you think there's another better way)

Reply:
Works great.